src/map/status_effect.h

```diff
@@ -717,12 +718,71 @@ enum EFFECT
     EFFECT_ELEMENTALRES_DOWN        = 802, // Elemental resistance down
-    // EFFECT_PLACEHOLDER              = 803 // Description
-    // 804-1022
+    // EFFECT_PLACEHOLDER              = 802 // Description
+    // 802-1022
+    EFFECT_INDI_REGEN               = 821,
+    EFFECT_INDI_POISON              = 822,
+    EFFECT_INDI_REFRESH             = 823,
+    EFFECT_INDI_HASTE               = 824,
+    EFFECT_INDI_STR                 = 825,
+    EFFECT_INDI_DEX                 = 826,
+    EFFECT_INDI_VIT                 = 827,
+    EFFECT_INDI_AGI                 = 828,
+    EFFECT_INDI_INT                 = 829,
+    EFFECT_INDI_MND                 = 830,
+    EFFECT_INDI_CHR                 = 831,
+    EFFECT_INDI_FURY                = 832,
+    EFFECT_INDI_BARRIER             = 833,
+    EFFECT_INDI_ACUMEN              = 834,
+    EFFECT_INDI_FEND                = 835,
+    EFFECT_INDI_PRECISION           = 836,
+    EFFECT_INDI_VOIDANCE            = 837,
+    EFFECT_INDI_FOCUS               = 838,
+    EFFECT_INDI_ATTUNEMENT          = 839,
+    EFFECT_INDI_WILT                = 840,
+    EFFECT_INDI_FRAILTY             = 841,
+    EFFECT_INDI_FADE                = 842,
+    EFFECT_INDI_MALAISE             = 843,
+    EFFECT_INDI_SLIP                = 844,
+    EFFECT_INDI_TORPOR              = 845,
+    EFFECT_INDI_VEX                 = 846,
+    EFFECT_INDI_LANGUOR             = 847,
+    EFFECT_INDI_SLOW                = 848,
+    EFFECT_INDI_PARALYSIS           = 849,
+    EFFECT_INDI_GRAVITY             = 850,
+    EFFECT_GEO_REGEN                = 851,
+    EFFECT_GEO_POISON               = 852,
+    EFFECT_GEO_REFRESH              = 853,
+    EFFECT_GEO_STR                  = 854,
+    EFFECT_GEO_DEX                  = 855,
+    EFFECT_GEO_VIT                  = 856,
+    EFFECT_GEO_AGI                  = 857,
+    EFFECT_GEO_INT                  = 858,
+    EFFECT_GEO_MND                  = 859,
+    EFFECT_GEO_CHR                  = 860,
+    EFFECT_GEO_FURY                 = 861,
+    EFFECT_GEO_BARRIER              = 862,
+    EFFECT_GEO_ACUMEN               = 863,
+    EFFECT_GEO_FEND                 = 864,
+    EFFECT_GEO_PRECISION            = 865,
+    EFFECT_GEO_VOIDANCE             = 866,
+    EFFECT_GEO_FOCUS                = 867,
+    EFFECT_GEO_ATTUNEMENT           = 868,
+    EFFECT_GEO_WILT                 = 869,
+    EFFECT_GEO_FRAILTY              = 870,
+    EFFECT_GEO_FADE                 = 871,
+    EFFECT_GEO_MALAISE              = 872,
+    EFFECT_GEO_SLIP                 = 873,
+    EFFECT_GEO_TORPOR               = 874,
+    EFFECT_GEO_VEX                  = 875,
+    EFFECT_GEO_LANGUOR              = 876,
+    EFFECT_GEO_SLOW                 = 877,
+    EFFECT_GEO_PARALYSIS            = 878,
+    EFFECT_GEO_GRAVITY              = 879,
     // EFFECT_PLACEHOLDER             = 1023 // The client dat file seems to have only this many "slots", results of exceeding that are untested.
```

Comment:
unrelated to this PR but: at some point we need to see what exactly happens if we go above this max index the dat has - will the packet even support higher values? will the client use them if given?

We have limited real estate for user, and we have to make use of it cautiously, but we may have more headroom if we can exceed the 1023 "slots" allotted by the dat file.

Reply:
well now i dont need any of these so thats should help the situation some.